docs/source/distributions/k8s/stack_run_config.yaml

```diff
@@ -0,0 +1,121 @@
+version: '2'
```

Comment:
What's specific about this file versus `llama_stack/templates/remote-vllm/run.yaml`? - symlink?

Reply:
Many changes -- there's safety but also Postgres as the store for everything.